tools/functional-tester/etcd-tester/cluster.go

```diff
@@ -110,8 +126,20 @@ func (c *cluster) Bootstrap() error {
 		}
 	}
 
-	stressers := make([]Stresser, len(clientURLs))
-	for i, u := range clientURLs {
+	var stressers []Stresser
+	if c.v2Only {
```

Comment:
if c.v20only is ture, we will go into this if statement and then we will continue the for loop at line 142. so we will start both v2 and v3. this is not what we want.